metaflow/plugins/cards/card_cli.py

```diff
@@ -0,0 +1,388 @@
+from metaflow.client import Task
+from metaflow import JSONType, namespace
+from metaflow.exception import CommandException
+import webbrowser
+import re
+import click
+import os
+import signal
+import random
+from contextlib import contextmanager
+from functools import wraps
+from metaflow.exception import MetaflowNamespaceMismatch
+from .card_datastore import CardDatastore, NUM_SHORT_HASH_CHARS
+from .exception import (
+    CardClassFoundException,
+    IncorrectCardArgsException,
+    UnrenderableCardException,
+    CardNotPresentException,
+)
+
+from .card_resolver import resolve_paths_from_task, resumed_info
+
+
+def serialize_flowgraph(flowgraph):
+    graph_dict = {}
+    for node in flowgraph:
+        graph_dict[node.name] = {
+            "type": node.type,
+            "box_next": node.type not in ("linear", "join"),
+            "box_ends": node.matching_join,
+            "next": node.out_funcs,
+            "doc": node.doc,
+        }
+    return graph_dict
+
+
+def open_in_browser(card_path):
+    url = "file://" + os.path.abspath(card_path)
+    webbrowser.open(url)
+
+
+def resolve_card(ctx, pathspec, hash=None, type=None, follow_resumed=True):
```

Comment:
Since we only have one card per task, remove hash and type. I would keep just pathspec and follow_resumed. Or, if you really want to keep them, put them last because I am not sure if that is how we will end up specifying them.

Reply:
We can have many cards given that users can create an arbitrary number of cards for a task using the `create` cli command.  Keeping these args is necessary because type and hash are necessary filters when many cards are created (Even in future). I can change the order